### DISCUSSION TITLE
How to contribute to GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by JonnyBDev** [community/community#27718] [GitHub Education] [answered]
Hello.

I've got a question but I couldn't find a answer in the community or in any docs.

The company I am working with provides a piece os software for IAAS. What would be the workflow to offer this software (licensed) to students via the GitHub Student Developer Pack like "SENDGRID" or "JETBRAINS"? 

Thanks for the answers :)

**Comment by MrSarno** [comment, marked as answer]
Hi @JonnyBDev,

If you use [this contact form](https://partner.github.com/apply) to apply as a GitHub Partner on behalf of your organisation, GitHub staff will review the application and, if successful, you can discuss your offer to make licenses available to those granted the GitHub Student Pack.
